system/Database/ResultInterface.php

```diff
@@ -13,10 +13,6 @@
 
 use stdClass;
 
-/**
- * @template TConnection of object|resource
- * @template TResult of object|resource
- */
```

Comment:
I removed `TConnection` and `TResult` because they are not used in this interface.
@paulbalandan Are you okay?

Reply:
I think I added those template types to help phpstan think that the interface is a generic and thus implementing classes should specify the specific types.

Reply:
Restored `@template` without types, because of https://github.com/vimeo/psalm/issues/2625